src/WSMan.ts

```diff
@@ -181,19 +186,24 @@ export class WSManMessageCreator {
    * @param method string - methods not covered by createCommonBody()
    * @param resourceUriBase string - URI address of the resource
    * @param wsmanClass string - name of WSMAN class being used
-   * @param data Object being converted into XML format
+   * @param data Array or Object being converted into XML format
    * @returns string
    */
-  createBody = (method: string, wsmanClass: string, data?: any): string => {
-    this.processBody(data)
+  createBody = (method: string, wsmanClass: string, data?: any | any[]): string => {
```

Comment:
since `any[]` is of type `any`. I don't think it is necessary to have both types especially since you are coercing is at an array anyway. Since you're doing this coercion, I'm not sure I follow the reason why you changed all the tests to pass in an array which essentially negates the coercion you did. I understand its for backwards compatibility -- but if that is the case typically you would leave all your existing tests as-is to ensure backwards compatability and only new tests would take advantage of allowing an array to be passed in.  When its time to move to vNext, that is the time all the tests would/should be changed.

Reply:
duly noted.  Will resolve in next refactoring